app/views/play/level/PlayLevelView.coffee

```diff
@@ -195,6 +196,10 @@ module.exports = class PlayLevelView extends RootView
     @controlBar.setBus(@bus)
     @initScriptManager()
 
+  onWorldNecessityLoadFailed: (resource) ->
+    @$el.find('.level-loading-goals, .tip, .load-progress').hide()
+    @$el.find('.could-not-load').show()
```

Comment:
Elements in a view should only be manipulated by their own view. In this case, the PlayLevelView is manipulating elements in LevelLoadingView. LevelLoadingView should have a function like this (maybe instead called `showCouldNotLoadError`), that PlayLevelView calls when it gets the error from LevelLoader.